From 73037c4ecd36eaf7ff35504f524344d1ab5a3f18 Mon Sep 17 00:00:00 2001
From: John Poet <jpoet@mythtv.org>
Date: Wed, 7 May 2014 09:44:18 -0600
Subject: [PATCH 3/9] Don't segfault if the screen has a disabled buttonlist,
 but the theme does not account for it.

Refs [eb17a56d1]
---
 mythtv/libs/libmythui/mythuibuttonlist.cpp |    1 +
 1 file changed, 1 insertion(+)

diff --git a/mythtv/libs/libmythui/mythuibuttonlist.cpp b/mythtv/libs/libmythui/mythuibuttonlist.cpp
index a2e00f1..95966a5 100644
--- a/mythtv/libs/libmythui/mythuibuttonlist.cpp
+++ b/mythtv/libs/libmythui/mythuibuttonlist.cpp
@@ -3439,6 +3439,7 @@ void MythUIButtonListItem::SetToRealButton(MythUIStateType *button, bool selecte
             state = "inactive";
             if (!button->GetState(state))
                 state = "active";
+            buttonstate = dynamic_cast<MythUIGroup *>(button->GetState(state));
         }
         else
             return;
-- 
1.7.10.2

